pandas/core/arrays/timedeltas.py

```diff
@@ -139,7 +140,7 @@ def _simple_new(cls, values, freq=None, dtype=_TD_DTYPE):
         result._freq = freq
         return result
 
-    def __new__(cls, values, freq=None):
+    def __new__(cls, values, freq=None, dtype=_TD_DTYPE):
```

Comment:
(why) was this change necessary?

Reply:
Because in the copy, from_factorized, concat_same_type methods we are passing dtype to this

Reply:
Whoops, I completely missed that the `dtype` argument wasn't present there before. I thought you were changing it from None to `_TD_DTYPE`.

Reply:
shouldn't this not still be passed as None and an if check inside? e.g.

```
if dtype is None:
     dtype = _TD_DTYPE
```

as seems passing None should error now